src/Vibe.C

```diff
@@ -311,7 +319,7 @@ Vibe::vibefilter(float data, fparams *ftype, int stage)
 };
 
 float
-Vibe::bjt_shape(float data)
+Vibe::bjt_shape(float data) const
```

Comment:
for my own education, is this const making it return a const (immutable) float? what is the motivation for this change?

Reply:
it means that a method does not modify any class member variables.

I've added this while debugging, to ensure that `vibefilter()` and `bjt_shape()` don't change the state. Both methods could actually be static, since they don't use any member vars.